openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -710,11 +710,19 @@ class LibraryBlockOlxView(APIView):
     @convert_exceptions
     def get(self, request, usage_key_str):
         """
+        DEPRECATED. Use get_block_olx_view() in xblock REST-API.
+        Can be removed post-Teak.
+        
         Get the block's OLX
         """
         key = LibraryUsageLocatorV2.from_string(usage_key_str)
+        version = request.query_params.get('version', None)
+
+        # Use the default `LatestVersion.AUTO` if `version` is None
+        version = xblock_api.string_to_version(version)
```

Comment:
You can remove `string_to_version` and undo these changes to the library REST API, now that we have the new XBlock REST API that supports versions.

Reply:
Updated https://github.com/openedx/edx-platform/pull/35932/commits/46ac9e2da497011bba8558679d890164c8948fb7